packages/build-utils/src/prerender.ts

```diff
@@ -3,6 +3,7 @@ import { Lambda } from './lambda';
 
 interface PrerenderOptions {
   expiration: number | false;
+  staleExpiration?: number;
```

Comment:
Let's break these `build-utils` changes out into their own PR so we can update independently